examples/decoding/decoding_csp_eeg.py

```diff
@@ -52,9 +52,6 @@
 montage = make_standard_montage('standard_1005')
 raw.set_montage(montage)
 
-# strip channel names of "." characters
-raw.rename_channels(lambda x: x.strip('.'))
-
```

Comment:
why?

Reply:
It's already fixed above in this example, this removed call does nothing